tools/automation/tests/main.py

```diff
@@ -33,7 +33,7 @@ def run_tests(modules, parallel, run_live, tests):
     test_paths = tests or [p for _, _, p in modules]
 
     display('Drive test by nosetests')
-    runner = get_nose_runner(parallel=parallel, process_timeout=3600 if run_live else 600)
+    runner = get_nose_runner(parallel=parallel, process_timeout=3600 if run_live else 1200)
```

Comment:
@fengzhou-msft do you know how to debug this script?

Reply:
I haven't worked with nose runner, ping @haroldrandom